### PR TITLE
adding new cards and removing carousel

### PR DESCRIPTION
**Pull request by prakhyatox**
Ticket : https://github.com/FAIRsharing/fairsharing.github.io/issues/2105

Adding new community champions logo card and removing carousel

### CODE REVIEW

**Comment by prakhyatox** [comment]
@allysonlister could you please share the content for the new cards which are placed by removing carousel.

https://deploy-preview-2124--fairsharing.netlify.app/

**Comment by allysonlister** [comment]
Thanks for this! I like the look of the two new circles for the Champions and the Adopters/Collaborators. Please can you let me know what you mean by the content for the new cards that you need? It looks like removing the carousel and just replacing it witht the new circle is good - did Susanna ask for something in particular while I was on leave?

**Comment by prakhyatox** [comment]
Screenshot 2023-09-06 at 16 32 04

Text for the highlighted section.
Yes, it was asked by Susanna

**Comment by allysonlister** [comment]
Thank you!

- [x] Please can the subject wheel be made a little bigger to better match the size of the assistant logo
DNcolksNJrDPUsiL

- [x] Please can the text for "Adopters and Collaborators" be changed to: "Working with our community to enable FAIR by promoting the value and the use of standards, databases and policies". However, if that doesn't fit on the two lines (as the text for the other circle does), let me know and I'll shorten it.

Thanks!

**Comment by allysonlister** [comment]
- [ ] Also, and I'm not 100% sure of this, it seems that the button+logo for the subject browser are centered relative to each other, but NOT centered relative to the background area.

**Comment by prakhyatox** [comment]
I have increased the browser subject logo size a bit bigger and updated the text

**Comment by prakhyatox** [comment]
> * [ ]  Also, and I'm not 100% sure of this, it seems that the button+logo for the subject browser are centered relative to each other, but NOT centered relative to the background area.

It won't be possible to same spacing for all three images because they all are of different dimension and they all are centrally aligned and appropriately spaced irrespective of screen dimension. 
If we decrease the spacing for subject logo to match of the FAIRassistant logo it will look something like this below

Screenshot 2023-09-07 at 13 50 46

It same vice-versa.

**Comment by allysonlister** [comment]
OK, so the text is over 3 lines even on my large window, so here's the updated text for Adopters and Collaborators:

"Learn about the people and communities who contribute to and advocate for FAIRsharing."

**Review by allysonlister — APPROVED** [review]
We're ready to go with these changes, thank you. 

Please can you split out the following changes into a separate ticket, as per Susanna's request?
- [ ] In the footer, change "Adopters" to "Adopters and Collaborators" and change the URL there to https://fairsharing.org/communities
- [ ] In the footer, remove the "Stakeholders" line 

Thanks!

File: src/components/Home/CommunityBlock.vue [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      >
+        <v-card
+          class="mx-auto block-category__card"
+          max-width="350"
+          height="350px"
+          width="350"
+        >
+          <div
+            :style="{ backgroundImage: 'linear-gradient(90deg, #e67e22, #ea862d, #ed8e38, #f19642, #f59e4c, #f8a657, #fcad61, #ffb56b),url(' + 'assets/Home/BlockHero/pattern-orange.png',backgroundSize:'100%',backgroundBlendMode:'color'}"
+            class="white--text d-flex flex-column justify-center block-category__card__gradiant__2"
```
Comment (knirirr) [inline src/components/Home/CommunityBlock.vue]:
```suggestion
            class="white--text d-flex flex-column justify-center block-category__card__gradient__2"
```

File: src/components/Home/CommunityBlock.vue [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+      >
+        <v-card
+          class="mx-auto block-category__card"
+          max-width="350"
+          height="350px"
+          width="350"
+        >
+          <div
+            :style="{ backgroundImage: 'linear-gradient(180deg, #7d7d7d, #868686, #8f8f8f, #989898, #a1a1a1, #aaaaaa, #b4b4b4, #bdbdbd),url(' + 'assets/Home/BlockHero/pattern-gray.png',backgroundSize:'300px',backgroundBlendMode:'lighten'}"
+            class="white--text d-flex flex-column justify-center  block-category__card__gradiant__1"
```
Comment (knirirr) [inline src/components/Home/CommunityBlock.vue]:
```suggestion
            class="white--text d-flex flex-column justify-center  block-category__card__gradient__1"
```

File: src/components/Home/CommunityBlock.vue [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    -moz-box-shadow: 0 1rem 2rem rgba(black, .15) !important;
+    -o-box-shadow: 0 1rem 2rem rgba(black, .15) !important;
+    &:hover {
+      transform: scale(1.05);
+      -moz-transform: scale(1.05);
+      -webkit-transform: scale(1.05);
+      -o-transform: scale(1.05);
+    }
+
+    &__gradiant__1 {
```
Comment (knirirr) [inline src/components/Home/CommunityBlock.vue]:
```suggestion
    &__gradient__1 {
```

File: src/components/Home/CommunityBlock.vue [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      height: 200px;
+      background: rgb(37, 52, 66);
+      background: linear-gradient(180deg, rgba(37, 52, 66, 1) 0%, rgba(39, 170, 225, 1) 100%);
+      background: -webkit-linear-gradient(180deg, rgba(37, 52, 66, 1) 0%, rgba(39, 170, 225, 1) 100%);
+      background: -o-linear-gradient(180deg, rgba(37, 52, 66, 1) 0%, rgba(39, 170, 225, 1) 100%);
+      background: -ms-linear-gradient(180deg, rgba(37, 52, 66, 1) 0%, rgba(39, 170, 225, 1) 100%);
+    }
+
+    &__gradiant__2 {
```
Comment (knirirr) [inline src/components/Home/CommunityBlock.vue]:
```suggestion
    &__gradient__2 {
```

**Review by knirirr — APPROVED** [review]
This should be fine (though gradiant should be changed to gradient).

File: src/components/Home/CommunityBlock.vue (hunk 4 above)
Comment (prakhyatox) [reply src/components/Home/CommunityBlock.vue]: Fixed

File: src/components/Home/CommunityBlock.vue (hunk 3 above)
Comment (prakhyatox) [reply src/components/Home/CommunityBlock.vue]: Fixed

File: src/components/Home/CommunityBlock.vue (hunk 2 above)
Comment (prakhyatox) [reply src/components/Home/CommunityBlock.vue]: Fixed

File: src/components/Home/CommunityBlock.vue (hunk 1 above)
Comment (prakhyatox) [reply src/components/Home/CommunityBlock.vue]: Fixed

**Pull request merged**